lib/virtfs/xfs/file_class.rb

```diff
@@ -0,0 +1,224 @@
+module VirtFS::XFS
+  class FS
```

Comment:
@roliveri but your protofs and nativefs code under the VirtFS use the full pathname.  So we need to pick one and be consistent.